src/main/java/nextstep/auth/AuthService.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.auth;
+
+import nextstep.support.MemberNotFoundException;
+import nextstep.member.MemberDao;
+import nextstep.support.UnauthorizedException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthService {
+
+    private final static JwtTokenProvider TOKEN_PROVIDER = new JwtTokenProvider();
+    private MemberDao memberDao;
+
+    public AuthService(MemberDao memberDao) {
+        this.memberDao = memberDao;
+    }
+
+    public void validatePassword(TokenRequest tokenRequest) {
+        if (memberDao.findByUsername(tokenRequest.getUsername())
+                .orElseThrow(MemberNotFoundException::new)
+                .checkWrongPassword(tokenRequest.getPassword())) {
+            throw new UnauthorizedException();
+        }
+    }
```

Comment:
로직을 작성할 때 재사용되지 않더라도 적절하게 지역변수를 사용하면 가독성에 도움이 될 수 있습니다!

현재 메서드의 이름은 validatePassword 인데, 유저의 `이름` 과 `패스워드` 를 검증하는 두 가지 일을 하고 있네요!
하나의 메서드에서 하나의 일을 할 수 있도록 개선해보면 좋을 것 같아요 :)

```suggestion
    public void validatePassword(TokenRequest tokenRequest) {
        Member member = memberDao.findByUsername(tokenRequest.getUsername())
                .orElseThrow(MemberNotFoundException::new);
        
        if (member.checkWrongPassword(tokenRequest.getPassword())) {
            throw new UnauthorizedException();
        }
    }
```